s4/clients/local.py

```diff
@@ -17,8 +17,8 @@
 logger = logging.getLogger(__name__)
 
 
-def get_local_client(target):
-    return LocalSyncClient(target)
+def get_local_client(target, state_folder):
```

Comment:
some tests in `test_local.py` would be ideal to show how `state_folder` affects the behaviour of the local client